contracts/migration/cypress/Aggregator/migration.lock

```diff
@@ -43,3 +43,8 @@
 20230925130908_UNI-USDT_register.json
 20230925130908_XRP-USDT_register.json
 20231022211021_JOY-USDT.json
+20231102141113_BORA-KRW.json
+20231102141113_KSP-KRW.json
+20231102141113_MBX-KRW.json
+20231102141113_WEMIX-USDT.json
+accountList.json
```

Comment:
@nick-bisonai Please remove this `accountList.json`

Reply:
removed it! thank you for pointing out